src/main/java/io/debezium/connector/yugabytedb/YugabyteDBSchema.java

```diff
@@ -410,13 +469,22 @@ protected void refreshSchemaWithTablet(TableId id, String tabletId) {
      */
     protected void buildAndRegisterSchemaForTablet(TableId id, String tabletId) {
         String lookupKey = getLookupKey(id, tabletId);
+        TableSchemaBuilder schemaBuilder;
+        if (config.isYSQLDbType()) {
+            schemaBuilder = getTableSchemaBuilder(config, valueConverter);
+        } else {
+            schemaBuilder = getTableSchemaBuilder(config, cqlValueConverter);
+        }
+
         Table table = tabletIdToTable.get(lookupKey);
-        TableSchemaBuilder schemaBuilder = getTableSchemaBuilder(config, valueConverter);
-        TableSchema schema = schemaBuilder.create(getSchemaPrefix(config.getLogicalName()), getEnvelopeSchemaName(table), table, config.getColumnFilter(), ColumnMappers.create(config), config.getKeyMapper());
 
+        TableSchema schema = schemaBuilder.create(getSchemaPrefix(config.getLogicalName()), getEnvelopeSchemaName(table), table, config.getColumnFilter(), ColumnMappers.create(config), config.getKeyMapper());
+      
         if (tableFilter.isIncluded(table.id())) {
             LOGGER.debug("Updating table schema with lookup key {}", lookupKey);
             tabletIdToTableSchema.put(lookupKey, schema);
+        } else {
+            LOGGER.warn("{} not included ",table.id());
```

Comment:
Perhaps we can be more informative in the log and say "{} is not included in the table filter list"

Btw, what is the case where we can hit this else block?

Reply:
@vaibhav-yb Basically, since the table filters for cql and sql were different I was hitting an issue where Schema was not getting built for CQL. I found this particular log helpful while debugging so decided to keep it as a warning. Do you want me to remove this?